### DISCUSSION TITLE
Loco position system  - crazyswarm does not detect the movement

### DISCUSSION THREAD

**Discussion by frulloa** [USC-ACTLab/crazyswarm#740] [Q&A]
Hello, please your kindly help. 

Unfortunately, when I run RVIZ the CF position just goes up and down (just moves on Z axis) (image below) even though I move the CF on X and Y axis too. So, the CF goes crazy with any test. **How can I solve this problem?** I followed the configuration steps of your [webpage](https://crazyswarm.readthedocs.io/en/latest/configuration.html) the hover_swarm.launch looks like this:
```
      motion_capture_type: none # one of none,vicon,optitrack,optitrack_closed_source,qualisys,vrpn
      object_tracking_type: "motionCapture" # one of motionCapture,libobjecttracker
      send_position_only: False # set to False to send position+orientation; set to True to send position only
      motion_capture_host_name: "vicon"
      # motion_capture_interface_ip: "" # optional for optitrack with multiple interfaces
      save_point_clouds: "/dev/null" # set to a valid path to log mocap point cloud binary file.
      print_latency: False
      write_csvs: False
      force_no_cache: False
      enable_parameters: True
      enable_logging: True
      enable_logging_pose: True
```
I am using Loco Position System. The LPS is setup with TDoA3. Furthermore, I test the system with bitcraze API, hence it is working. 

![Screenshot from 2023-05-16 16-11-16](https://github.com/USC-ACTLab/crazyswarm/assets/78211872/1eed6fb0-6be5-43bc-bfc7-2de6735a1e62)

**Comment by whoenig** [comment]
We don't really support Crazyswarm1 anymore - the successor is [Crazyswarm2](https://imrclab.github.io/crazyswarm2/).

You could look at `ros topic echo /cf4/pose` to see the actual raw data that you get from the drone. There is not that much Crazyswarm is doing (compared to the Bitcraze API) when LPS is used.

**Reply by frulloa** [reply]
Ok, thank you.
